Gemfile

```diff
@@ -198,3 +198,5 @@ group :development do
   gem "derailed_benchmarks", "~> 2.1"
   gem "bullet"
 end
+
+gem "inertia_rails", "~> 3.10"
```

Comment:
I assume you used `bundle add` to install this, but we prefer to list the gems in alphabetical order in the Gemfile. Can you move it accordingly?

Reply:
done